assets/js/components/icons/delete.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import { Icon } from '@wordpress/components';
+
+// This is named `delete` as it is the `delete_outline` icon from Material.
+// https://material.io/resources/icons/?icon=delete_outline&style=baseline
+// We are using it as `trash` or `trashcan` or `remove-item`.
+export default () => (
```

Comment:
Not sure on the name for this file.  I think we should be specific and call it `trash_outline` or something like that.

Reply:
Went with `IconTrash` - that's the simplest/shortest, and is a good combo of semantic (we use this for buttons which trash items) and visual (the icon is a trashcan). Didn't add `-outline` because that feels like an implementation detail (we only use outline version so far). 89d77ec3a1e088bf3064122e698a81a5b6a95bae